src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SEPARATOR = ",";
+    private int buyRep = 0;
+    private int supplyRep = 0;
+    private StringBuilder stringBuilder = new StringBuilder();
+    private int rows = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        WorkWithFile statistics = new WorkWithFile();
+        statistics.readFromFile(fromFileName);
```

Comment:
всередині класу тоді не потрібно створювати об'єкт класу щоб звернутись до методу 
```suggestion
        String[] content = readFromFile(fromFileName);
```